### PR TITLE
Added "YAML" section to "Importing Data" page

### PR DESCRIPTION
**Pull request by CodingCarter**

### CODE REVIEW

**Comment by CodingCarter** [comment]
Note: I could just destructured the product variable and change it to let { products } = the_value, but I'll let the mods decide that because it may seem confusing to some people.

File: docs/fetching-data.md [hunk 1]
```diff
@@ -65,15 +65,39 @@ module.exports = function (api) {
...
+
+
+Here is an example `gridsome.server.js` file that imports the yaml files:
+
+```js
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+let fileContents = fs.readFileSync('./src/data/products.yaml', 'utf8');
+let products = yaml.safeLoad(fileContents).products;
```
Comment (CodingCarter) [inline docs/fetching-data.md:80]: Could be let { products } = yaml.safeLoad(fileContents);

File: docs/fetching-data.md [hunk 2]
```diff
@@ -65,15 +65,39 @@ module.exports = function (api) {
 *..contribute*
 
 ### YAML
-*..contribute*
+Import data from any yaml file to the GraphQL data layer with the [Data store API](/docs/data-store-api/). To use the API you need a `gridsome.server.js` file in the root folder of your Gridsome project.  First, though, you need to install the node packages `fs` and `js-yaml` into your project. Run the following command to install them:
+
+`npm install fs js-yaml --save`
```
Comment (mtlynch) [inline docs/fetching-data.md:70]: Should be `--save-dev` since we need it only at build time. By runtime, Gridsome has already converted the YAML to JSON.

**Comment by mtlynch** [comment]
Merged in #491

**Pull request closed without merging**